examples/EA/MOO/MOCMAExperiment.tpp

```diff
@@ -52,7 +52,8 @@ double hypervolume( Solution const& solution){
 	RealVector referencePoint(2,11);
 	//instance of the hypervolume calculator
 	HypervolumeCalculator hypervolume;
-	return hypervolume(PointExtractor(),solution,referencePoint);
+	auto toPoints = [](typename Solution::const_reference point){return point.value;};

```

Comment:
I cannot find the definition of the PointExtractor, but I am asking myself how this line differs, and whether the PointExtractor can/should be used instead.